minifi/NOTICE

```diff
@@ -0,0 +1,7 @@
+Apache NiFi - MiNiFi
```

Comment:
this file should go away completely / This is about the source NOTICE and thus whatever is in here that isn't in the NiFI/NOTICE should get added there and this file removed.